<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGraphs</class>
 <widget class="QWidget" name="TestGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
</string>
  </property>
  <widget class="QCustomPlot" name="customPlot_histogram" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value_1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_normalize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot_time_series" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
